community/s-nail/APKBUILD

```diff
@@ -15,7 +15,8 @@ subpackages="$pkgname-doc"
 source="https://ftp.sdaoden.eu/s-nail-$pkgver.tar.xz"
 builddir="$srcdir/"$pkgname-$pkgver
 
-build() {
+prepare() {
```

Comment:
Is there a reason you are moving this to prepare, rather than leaving this in build?

Reply:
I've done this for testing the build more quickly, but i'll revert this change.